spec/factories/plates.rb

```diff
@@ -1,4 +1,27 @@
 FactoryBot.define do
   factory :plate do
+    factory :plate_with_wells do
+      transient do
+        well_count { 1 }
+      end
+  
+      after :create do |plate, options|
+        options.well_count.times do |i|
+          create(:well, position: "A#{i}", plate: plate)
+        end
+      end
+    end
+
+    factory :plate_with_ont_samples do
```

Comment:
Good job getting these in here.  When I created the separate file I didn't know this was how it was supposed to be arranged.